src/js/include/llm_pipeline/llm_pipeline_wrapper.hpp

```diff
@@ -1,5 +1,6 @@
 #pragma once
 
+#include <future>
```

Comment:
I think it's better to keep in header only necessary files and move it down into the source file

Reply:
Thank you, It has fixed